src/pages/home/home.tsx

```diff
@@ -1,20 +1,60 @@
-import React from "react";
-import { MainBanner } from "uikit/molecules/MainBanner";
-import { MovieList } from "uikit/molecules/MovieList";
+import * as React from "react";
+
+import { DeleteModal } from "uikit/molecules/DeleteModal";
+import { EditModal } from "uikit/molecules/EditModal";
```

Comment:
I'd recommend to create index.ts file in uikit/molecules folder and export all these molecules from this file to have just 1 import here:
import { DeleteModal, EditModal, ...etc } from 'uikit/molecules'

Reply:
I have no idea why I didn't do it before.